configure.ac

```diff
@@ -1266,6 +1266,29 @@ else
     fi
 fi
 
+AC_MSG_CHECKING([for std::system])
```

Comment:
Already present in https://github.com/dashpay/dash/pull/4864 as https://github.com/dashpay/dash/pull/4864/commits/6627bc968873a86e46c2eb72f64a0d30c9c32f11.

Reply:
removed